configuration/app-settings.md

```diff
@@ -192,4 +192,31 @@ Replications are defined under the `replications` key as an array of replication
 |`fromSuffix`|The suffix of the source table(s). Regular expression may be used.|yes|
 |`toSuffix`|The suffix of the target table.|yes|
 |`text_expression`|Any valid text expression. For more information, see [LaterJS](https://bunkat.github.io/later/parsers.html#text)|no if `cron` provided|
-|`cron`|Any valid Cron expression. For more information, see [LaterJS](https://bunkat.github.io/later/parsers.html#cron)|no if `text_expression` provided|
\ No newline at end of file
+|`cron`|Any valid Cron expression. For more information, see [LaterJS](https://bunkat.github.io/later/parsers.html#cron)|no if `text_expression` provided|
+
+### Configuring place hierarchy
+
+From 3.6.0 it is possible to configure what types of places and people are available by modifying the `contact_types` array in the app settings. Each type has the following properties.
+
+|Property|Description|Required|
+|-------|---------|----------|
+| `id` | String identifier for the type. At some times this will be used to sort the contacts in the UI so it is recommended to using a number prefix with gaps between numbers, eg: `10-district`, `20-region`, etc. | Yes. |
+| `name_key` | The translation key used for the title for the contact profile. | No, defaults to 'contact.profile'. |
+| `group_key` | The translation key used for the title of a list of contacts of this type. | Yes. |
+| `create_key` | The translation key used on the button for creating new contacts of this type. | Yes. |
+| `edit_key` | The translation key used on the button for editing contacts of this type. | Yes. |
+| `primary_contact_key` | The translation key used to identify a person as the primary contact of contacts of this type. | No, defaults to 'Primary contact'. |
+| `parents` | An array of strings of IDs of parent types. If more than one then this type can appear in different places in the hierarchy. If more than one type lists the same type as a parent then a user will get a dropdown of places to create. If no parents then contacts of this type will be at the top of the hierarchy and cannot be added as a child of any contact. | No, defaults to no parents. |
```

Comment:
Is it supported to have a type which can be at the "top of hierarchy" and also be under another contact type (eg. clinic)? If so, how is this expressed? `parent: [undefined, "clinic"]`?

Reply:
That is not currently supported. Can you think of a use case where this would be desirable?

Reply:
No idea. Just checking for the parsing in `move-contacts`